core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/gather.kt

```diff
@@ -18,6 +25,115 @@ import kotlin.reflect.typeOf
 
 // region gather
 
+/**
+ * Reshapes the [DataFrame] by gathering specified [\columns] into two columns: keys and values
```

Comment:
no (small) header? :/ okay we don't necessarily need them

Reply:
Haven't we agreed that we don't place header in kdocs?